Crashlytics/Crashlytics/Helpers/FIRCLSDefines.h

```diff
@@ -23,9 +23,8 @@
 // These macros generate a function to force a symbol for the containing .o, to work around an issue
 // where strip will not strip debug information without a symbol to strip.
 #define DUMMY_FUNCTION_NAME(x) CONCAT(fircls_strip_this_, x)
-#define INJECT_STRIP_SYMBOL(x)        \
-  void DUMMY_FUNCTION_NAME(x)(void) { \
-  }
+#define INJECT_STRIP_SYMBOL(x) \
+  void DUMMY_FUNCTION_NAME(x)(void) {}
```

Comment:
This was done to silence a formatting [error](https://github.com/firebase/firebase-ios-sdk/actions/runs/3888342893/jobs/6635556377). I wonder if this will occur more frequently as we diverge further from `master`.